### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD013.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#19] [General]
# https://blog.3place.cn/3place/fsd/FSD013.html

https://blog.3place.cn/3place/fsd/FSD013.html

**Comment by picklecai** [comment]
梅菲斯特虽然捏造事实，但他仍然需要出证明。

**Comment by picklecai** [comment]
玛尔太丈夫的故事编得一波三折，是为了让小姑娘动摇么？
